report_custom_filename/README.rst

```diff
@@ -22,6 +16,15 @@ In case of trouble, please check there if your issue has already been reported.
 If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
 `here <https://github.com/OCA/reporting-engine/issues/new?body=module:%20report_custom_filename%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
 
+Usage
+=====
+
+To use this module, you need to:
```

Comment:
Remove this sentence. It's intended as template to explain the usage, not as an introduction to the "Try me on runbot" button